lib/eex/test/eex_test.exs

```diff
@@ -369,13 +369,13 @@ defmodule EExTest do
     test "evaluates the source" do
       filename = Path.join(__DIR__, "fixtures/eex_template.eex")
       result = EEx.eval_file(filename)
-      assert result == "foo bar.\n"
+      assert result |> String.trim_trailing() == "foo bar." |> String.trim_trailing()
```

Comment:
Instead of String.trim_trailing, let's do `String.replace(string, "\r\n", "\n")`. Can you also please move the string.replace call to a separate private function so we can re-use it through the file?